doc/man/salt.7

```diff
@@ -198363,7 +198363,7 @@ installed pip is new enough.
 .UNINDENT
 .INDENT 0.0
 .TP
-.B salt.modules.pip.list_all_versions(pkg, bin_env=None, include_alpha=False, include_beta=False, include_rc=False, user=None, cwd=None)
+.B salt.modules.pip.list_all_versions(pkg, bin_env=None, include_alpha=False, include_beta=False, include_rc=False, user=None, cwd=None, index_url=None)
```

Comment:
It's totally fine that these changes are here, but I wanted to let you know that we automatically generate the manpages for each release, so you don't need to update these in the future. :)

Reply:
😸